module-1/lab-api-scavenger-game/your-code/Challenge 2.ipynb

```diff
@@ -207,79 +207,887 @@
   },
```

Comment:
There is something wrong in your code. You should get something like 22 commits, you get 6 instead. 
Take a look on it and get back to me tomorrow for any question!!